src/Storage/N3O.Umbraco.Storage/Controllers/StorageController.cs

```diff
@@ -48,22 +44,16 @@ public class StorageController : ApiController {
 
     [HttpPost("upload/{folderPath}")]
     public async Task<ActionResult<StorageToken>> Upload(string folderPath, [FromForm] UploadReq req) {
```

Comment:
@talhamalik4025 Controller actions such as this which take a request model but don't use mediator, don't we need to change the base class to ValidatingApiController and begin the action method by invoking validation of the model?